frontend/app/src/mixins/backend-mixin.ts

```diff
@@ -32,10 +31,6 @@ export default class BackendMixin extends Vue {
   async mounted() {
     await this.load();
     this.loaded();
-    const loglevel = this.options.loglevel;
```

Comment:
after removing this will the options loglevel still be considered on the electron app?

Reply:
Ok it seems to work fine

Reply:
@lukicenturi so the UI updates properly if I pass a config file [loglevel](https://rotki.readthedocs.io/en/latest/usage_guide.html#set-the-backend-s-arguments)

![image](https://user-images.githubusercontent.com/2269732/152167901-3e6c42e7-d309-4ed0-91ba-18de7d8fc996.png)

but frontend remains to WARN
![image](https://user-images.githubusercontent.com/2269732/152167964-8f2e0f5c-24f7-404b-a183-70029aca6ffa.png)

We still need to keep this behavior when we are packaged using electron. When there is a config file that sets loglevel, the loglevel from the config file should take priority and cannot be modified.

Also in electron the user can modify the loglevel and this takes priority, so at least for electron we shhou keep this behavior here and only set from the one in the backend in docker

Reply:
I guess no. The options log level is used to restart the backend. And the log level value is always taken from the API at `store.ts`. So I think they will be always the same(?) @kelsos